### ISSUE TITLE
Add "Cancel current order" response handler

### ISSUE THREAD

**Issue by ro31337** [ro31337/libretaxi#208]
"Cancel current order" response handler should cancel current order for specified passenger. It also includes notifying drivers who were notified already about this order (probably functionality for later story, because we don't notify drivers now).

So minimum implementation for this order should just cancel the order, set it's `status` to `cancelled` (status should be introduced for `Order`  before, if it's not there). And it should create call for phantom menu action called `OrderCancelled` that will display the text and redirect passenger to their index page.

Keep in mind that we also want to allow users to cancel their orders only once in 15 minutes (can be a part of another story).
